test/utils/image/manifest.go

```diff
@@ -151,6 +151,8 @@ const (
 	None ImageID = iota
 	// Agnhost image
 	Agnhost
+	// AgnhostNew image
+	AgnhostNew
```

Comment:
```suggestion
	AgnhostNMinus1
```

Reply:
Remove the httpd images from the manifest as well, and then fix them in all other places.